diff --git a/node_modules/react-native/gradle/libs.versions.toml b/node_modules/react-native/gradle/libs.versions.toml
index 2f9c466..2398e36 100644
--- a/node_modules/react-native/gradle/libs.versions.toml
+++ b/node_modules/react-native/gradle/libs.versions.toml
@@ -28,7 +28,7 @@ javax-inject = "1"
 jsc-android = "2026004.0.1"
 jsr305 = "3.0.2"
 junit = "4.13.2"
-kotlin = "2.1.20"
+kotlin = "2.3.20"
 ktfmt = "0.22.0"
 mockito = "3.12.4"
 mockito-kotlin = "3.2.0"
